addon/components/link-building-modal.js

```diff
@@ -4,5 +4,13 @@ import BuildingWork from "ember-ebau-gwr/models/building-work";
 
 export default class LinkBuildingModalComponent extends Component {
   @tracked buildingWork = new BuildingWork();
-  kindOfWorkOptions = BuildingWork.kindOfWorkOptions;
+  // Remove the state "Neubau" from the array since you cant link
+  // existing buildings as "Neubau".
+  kindOfWorkOptions = BuildingWork.kindOfWorkOptions.slice(1);
+
+  constructor(...args) {
+    super(...args);
+    this.buildingWork = new BuildingWork();
+    this.buildingWork.kindOfWork = this.kindOfWorkOptions[0];
+  }
```

Comment:
Yes i had the same thought but didn't know a not to expensive way.
Since i really dont want to duplicate values i would have to refactor the array to a object which, for a quick change is maybe a bit risky.